src/TestxServer.ts

```diff
@@ -227,4 +227,20 @@ export class TestxServer {
 
     return this.client.getMutations();
   }
+
+
+    /**
+   * Get the generated client subscriptions.
+   * @return {Object} An object containing the subscriptions as properties
+   */
+  public getSubscriptions(): { [name: string]: string } {
```

Comment:
We should add the method to the `TestxApi` and `TestxDirector` 
https://github.com/aerogear/graphql-testx/blob/cf9915f72ba4614b14639e2d6f2e4c16fbc63b53/src/TestxApi.ts#L7

Reply:
Sure, I'm going to update

Reply:
It's because I've merged the Director PR :)